### PR TITLE
feat(core): export tx Enum Type from core

### PR DESCRIPTION
**Pull request by alebusse**
Re-export accountLib imported transaction types

[stlx-3452](https://bitgoinc.atlassian.net/browse/STLX-3452)

### CODE REVIEW

**Review by rliberal-simtlix — APPROVED** [review]

**Review by CapnMigraine — DISMISSED** [review]
lgtm, but I'd like to have @tylerlevine's review as well.

**Review by tylerlevine — CHANGES_REQUESTED** [review]
Why do we need to re-export account lib types?

File: modules/core/src/v2/types.ts [hunk 1]
```diff
@@ -14,3 +16,13 @@ export interface RequestTracer {
   inc(): void;
   toString(): string;
 }
+
+
+export class Types {
```
Comment (rliberal-simtlix) [inline modules/core/src/v2/types.ts:21]: Talked with Tyler, better if we move this to basecoin
https://github.com/BitGo/BitGoJS/blob/db79e9500298563586a1febe1820ed245dda8fd7/modules/core/src/v2/baseCoin.ts#L4

**Comment by tylerlevine** [comment]
Yeah, probably this is all we need to do in `core`:
```
diff --git a/modules/core/src/v2/baseCoin.ts b/modules/core/src/v2/baseCoin.ts
index 2a25d9f4..e58855a2 100644
--- a/modules/core/src/v2/baseCoin.ts
+++ b/modules/core/src/v2/baseCoin.ts
@@ -12,6 +12,10 @@ import { NodeCallback } from './types';
 import { RequestTracer } from './internal/util';
 const co = Bluebird.coroutine;
 
+// re-export account lib transaction types
+import { BaseCoin as AccountLibBasecoin } from '@bitgo/account-lib';
+export type TransactionType = AccountLibBasecoin.TransactionType;
+
 import { Wallet } from './wallet';
 import { Wallets } from './wallets';
 import { Markets } from './markets';
```

Then we can use it in the client by doing this:
```
import { BaseCoin } from 'bitgo';
// use BaseCoin.TransactionType
```

**Review by tylerlevine — APPROVED** [review]

**Review by rliberal-simtlix — APPROVED** [review]

**Review by CapnMigraine — APPROVED** [review]

**Pull request merged**